practice/caesar/site.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**Returns the ASCII position of the inputted character.
```

Comment:
JSDoc style has the first line on the next line after `/**`.

If you install the [docblockr](https://atom.io/packages/docblockr) Atom package, it will help out with formatting those.